src/email/aws.email.service.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  SESClient,
+  SendEmailCommand,
+} from "@aws-sdk/client-ses";
+import { Email } from "./dto/email.dto";
+import { BaseEmailService } from "./interfaces/email.service.interface";
+import config from "./../utils/config";
+import { InjectMapper } from "@automapper/nestjs";
+import { Mapper } from "@automapper/core";
+
+export class AwsEmailService extends BaseEmailService {
+  private sesClient: SESClient;
+
+  constructor(@InjectMapper() private readonly mapper: Mapper) {
```

Comment:
We can get configurations as a AwsEmailConfig class in constructor. We shouldn't reach config directly from here. Instead, we can set this configurations while importing AwsEmailService provider in EmailModule.